client/state/happychat/selectors/is-presales-zendesk-chat-available.js

```diff
@@ -0,0 +1,11 @@
+import 'calypso/state/happychat/init';
+
+/**
+ * Returns if presales zendesk chat is available.
+ *
+ * @param   {object}  state  Global state tree
+ * @returns {boolean}        true, when presales_zendesk is available
+ */
+export default function isPresalesZendeskChatAvailable( state ) {
+	return state.happychat?.user?.availability?.presale_zendesk ?? false;
```

Comment:
I'd expect `state.happychat.user.availabiity` to always be there, since we import the reducer dynamically above on the first line. That being said, the optional chaining will be necessary only for `availability` because it can be `null` as per the default reducer value. 

```suggestion
	return state.happychat.user.availability?.presale_zendesk ?? false;
```